src/backend/expungeservice/models/helpers/record_summarizer.py

```diff
@@ -53,15 +54,27 @@ def summarize(record) -> RecordSummary:
         for county, balance in county_balances.items():
             county_balances_list.append(CountyBalance(county, round(balance, 2)))
         total_charges = len(record.charges)
-        eligible_charges = [
+        eligible_charges_now = [
             c.name
             for c in record.charges
             if c.expungement_result.charge_eligibility.status == ChargeEligibilityStatus.ELIGIBLE_NOW
         ]
+        eligible_charges_by_date = [["now",eligible_charges_now]]
+        will_be_eligible_charges : Dict[date, List[str]] = {}
+        for charge in record.charges:
+            if charge.expungement_result.charge_eligibility.status == ChargeEligibilityStatus.WILL_BE_ELIGIBLE:
+                date_eligible = charge.expungement_result.time_eligibility.date_will_be_eligible
+                charge_name = charge.name.split("(")[0]
```

Comment:
Maybe call it `charge_short_name` to make it clear you're just trying to shorten the name for aesthetic reasons.